quinn-proto/src/connection/mod.rs

```diff
@@ -2520,13 +2530,13 @@ where
 
     /// Validate transport parameters received from the peer
     fn validate_params(&mut self, params: &TransportParameters) -> Result<(), TransportError> {
-        if self.side.is_client() && self.orig_rem_cid != params.original_connection_id {
-            debug!(
-                "original connection ID mismatch: expected {:x?}, actual {:x?}",
-                self.orig_rem_cid, params.original_connection_id
-            );
-            return Err(TransportError::TRANSPORT_PARAMETER_ERROR(
-                "original CID mismatch",
+        if Some(self.orig_rem_cid) != params.initial_source_connection_id
```

Comment:
The transport parameters are named to exactly match the spec, which I'm inclined to stick with. I could align everything else more closely with those, but I don't know that I want to be that verbose for *every* mention of a CID internally, and there'll be some inconsistency regardless otherwise.

Reply:
It's a reasonable argument -- but we don't stick to the exact spec wording everywhere else (at least these abbreviations that we use we invented, IIRC). Why do you think the `TransportParameters` should be different?

Reply:
Slept on this and decided contracting the names here makes sense, since there's not really any realistic danger of confusion.